index.html

```diff
@@ -33,6 +33,293 @@
+      defined in the Web IDL specification [[!WEBIDL]], as this specification uses that
+      specification and terminology.
+    </p>
+  </section>
+  <section>
+    <h2>
+      Example
+    </h2>
+    <p>
+      The following example demonstrates a request for all
+      <a data-cite="screen-capture#dfn-monitor">monitors</a>
+      using the
+      <code>navigator.mediaDevices.getAllScreenMedia</code>
+      method defined in this document.
+    </p>
+    <pre class="example highlight">
+      try {
+        const mediaStreams = await navigator.mediaDevices.getAllScreenMedia();
+        mediaStreams.forEach((mediaStream, index) => {
+          videoElements[index].srcObject = mediaStream;
+        })
+      } catch (e) {
+        console.log('Unable to acquire screen captures: ' + e);
+      }
+    </pre>
+  </section>
+  <section id="terminology">
+    <h2>
+      Terminology
+    </h2>
+    <p>
+      This document uses the definition of {{MediaStream}}, {{MediaStreamTrack}}
+      from [[!GETUSERMEDIA]], <a data-cite="url#concept-url-origin">origin</a> from [[!url]],
+      {{ScreenDetailed}} from [[!window-placement]],
+      and <a data-cite="screen-capture#dfn-display-surface">monitor</a>
+      from [[!screen-capture]].
+    </p>
+  </section>
+  <section id="capture_multiple_display_surfaces">
+    <h2>
+      Capturing multiple monitors
+    </h2>
+    <p>
+      Capture of all <a data-cite="screen-capture#dfn-monitor">monitors</a>
+      is enabled through the addition of a new
+      {{MediaDevices/getAllScreenMedia}} method on the {{MediaDevices}}
+      interface.
+    </p>
+    <section>
+      <h2>
+        <dfn>MediaDevices</dfn> Additions
+      </h2>
+      <pre class="idl">
+          partial interface MediaDevices {
+            Promise&lt;sequence&lt;MediaStream&gt;&gt; getAllScreenMedia();
+          };
+        </pre>
+      <dl data-link-for="MediaDevices" data-dfn-for="MediaDevices" class="methods">
+        <dt>
+          <dfn>getAllScreenMedia</dfn>
+        </dt>
+        <dd>
+          <p>
+            The <a href="#dfn-user-agent">user agent</a> MUST return a list of {{MediaStream}}
+            objects, where each stream belongs to exactly one of the
+            <a data-cite="screen-capture#dfn-monitor">monitors</a> attached to the device.
+          </p>
+          <p>
+            {{PermissionState/"granted"}} permissions cannot be persisted.
+          </p>
+          <p>When the {{MediaDevices/getAllScreenMedia()}}
+            method is called, the <a href="#dfn-user-agent">user agent</a> MUST run the following
+            steps:</p>
+          <ol>
+            <li>
+              <p>Let <var>p</var> be a new promise.</p>
+            </li>
+            <li>
+              <p>Run the following steps in parallel:</p>
+              <ol>
+                <li>
+                  <p>Optionally, e.g., for security reasons, or due to platform
```

Comment:
I'd mention platform limitations for now. I don't have a security reason in mind right now, but I can easily think of platform limitations (too many concurrent captures).